src/PrestaShopBundle/Controller/Admin/WarehouseController.php

```diff
@@ -70,7 +70,7 @@ public function refreshProductWarehouseCombinationFormAction($idProduct)
             $this->get('prestashop.adapter.data_provider.tax'),
             $this->get('router')
         );
-        $allFormData = $modelMapper->getFormData();
+        $allFormData = $modelMapper->getFormData($product);
```

Comment:
:scream:

Reply:
:warning: The method`getFormData` of modelMapper has a missing parameter :(

I asked a exploratory analysis from QA